algorithms/Frequent_Pattern_Mining/fpm/tg_fpm.gsql

```diff
@@ -0,0 +1,147 @@
+CREATE QUERY tg_fpm(STRING unify_vertex_type, STRING item_vertex_type, STRING connect_edge, DOUBLE support, 
+                    UINT min_length, UINT lim = 10000, INT output_limit = -1, BOOL spit_to_file = FALSE,
+                    STRING file_path = "/home/tigergraph/tg_query_output.csv") { 
+  /*
+  Schema examples: user - (buys) - product
+                   user - (visits) - page
+                   user - (rates) - product
+
```

Comment:
If this algorithm follows the work of a published paper, it is good to cite the paper in the comments.

Reply:
added reference